Library/Homebrew/dev-cmd/bottle.rb

```diff
@@ -13,7 +13,7 @@
 
 BOTTLE_ERB = <<-EOS.freeze
   bottle do
-    <% if !["#{HOMEBREW_BOTTLE_DEFAULT_DOMAIN}/bottles", "https://homebrew.bintray.com/bottles"].include?(root_url) %>
+    <% if !root_url.start_with?(HOMEBREW_BOTTLE_DEFAULT_DOMAIN) %>
```

Comment:
What's this bit for? Not disagreeing just curious.

Reply:
This code has a purpose. Part of the macOS bottling pipeline runs on Linux, where `HOMEBREW_BOTTLE_DEFAULT_DOMAIN` is `https://linuxbrew.bintray.com/bottles`. We can end up with the opposite problem, where we'd have `root_url "https://homebrew.bintray.com/bottles"` in bottles.

Reply:
> This code has a purpose. Part of the macOS bottling pipeline runs on Linux, where HOMEBREW_BOTTLE_DEFAULT_DOMAIN is https://linuxbrew.bintray.com/bottles.

It is no longer in this case as we now set `HOMEBREW_FORCE_HOMEBREW_ON_LINUX` in brew test-bot.

Further, reverting this would also help to remove superfluous `root_url` in `linuxbrew/xorg` and `linuxbrew/extra`.

Reply:
Fine with me if/when it's fine with @sjackman.

Reply:
Fine with me then. Thanks, @xu-cheng.